### DISCUSSION TITLE
HELP Media Player card configuration

### DISCUSSION THREAD

**Discussion by SylvainDx** [Clooos/Bubble-Card#954] [Questions about config, custom styles and templates] [answered]
Hello,

I'm struguling with the Media Player card and how make it look like the screenshot in the presentation
![image](https://github.com/user-attachments/assets/d35bbe2e-4299-462a-b0db-badb5c0b7879)

especially the second line.
I can't manage to make the slider and have trouble to find how set the action for the mute command ... :(

thank you for your assistance !

**Comment by Clooos** [comment, marked as answer]
Hi, just press the volume icon, it will show the slider and mute button 🙂

**Comment by SylvainDx** [comment]
I'll be damned ! 🤣
Is it possible to have the card on this state by default ?

I found some style option to display the slider 
```
  .bubble-volume-slider.is-hidden {
  opacity: 100 !important;
  pointer-events: auto;
}
```
But I never understound how to format the tap action to have the mute command to work

**Reply by Clooos** [reply]
If you just want a volume slider, create a slider button then add your media player in it, this is the best way. And if you want to add a mute button to it, just create a sub-button with the tap action set to media_player.volume_mute, then don't forget to add the data part in YAML.

https://www.home-assistant.io/integrations/media_player/#action-media_playervolume_mute

**Reply by SylvainDx** [reply]
that's the data part I don't understand :( i'm sorry

**Reply by Clooos** [reply]
I just did a research on Google and this should helps you:

https://community.home-assistant.io/t/media-player-volume-mute-toggle/302265

**Reply by SylvainDx** [reply]
Ok I understand now why it wasn't obvious ....

Thank you so much !

**Reply by Clooos** [reply]
Yes indeed, it's was less obvious than I thought!

Glad that you make it works now 👌
